readthedocs/builds/urls.py

```diff
@@ -1,6 +1,6 @@
 """URL configuration for builds app."""
 
-from django.conf.urls import re_path
```

Comment:
Some of this PR seems to remove re_path, but this just changes the import?

Reply:
Yes. `re_path` was moved around and it's now under a different import path.

The usage of `re_path` here cannot be changed by `path` directly because we have non-simple/standard regex that django-upgrade can detect and adapt to just `path`. That's my bet, tho.